packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -955,46 +964,24 @@ describe('Checkout', () => {
             });
         });
 
-        describe('dialogMessage ::', () => {
```

Comment:
Removed `dialogMessage` and `alertMessage` tests as they are duplicated in other tests on lines 902 and 931